app/src/main/kotlin/com/wire/android/ui/common/Logo.kt

```diff
@@ -19,8 +20,8 @@ fun Logo() {
         contentDescription = stringResource(id = R.string.app_logo_description),
         contentScale = ContentScale.Fit,
         modifier = Modifier
-            .width(80.dp)
-            .height(30.dp)
-            .padding(8.dp)
+            .padding(paddingValues = PaddingValues(horizontal = 8.dp, vertical = 32.dp))
+            .width(72.dp)
+            .height(23.dp)
```

Comment:
move to dimensions, also 23.dp is afunny size :P maybe 24.dp would make more sense as it is multiplier of 8 :P

Reply:
Those are the dimensions provided by the design specs 🙈 but will bump them to prettier numbers 😉